src/ecmech/ECMech_util.h

```diff
@@ -1192,8 +1190,8 @@ namespace ecmech {
             mtanSD(jSvec, 5) = C(jSvec, 2) * sqr2;
          }
       }
-   } // mtan_conv_sd_svec
-
+   } // mtan_conv_sd_
```

Comment:
Why the change here? -- It does not look like the function name changed.

Reply:
Must have accidentally deleted this I'll revert the change.